src/win/winapi.c

```diff
@@ -63,6 +64,10 @@ void uv_winapi_init(void) {
     uv_fatal_error(GetLastError(), "GetModuleHandleA");
   }
 
+  pRtlGetVersion = (sRtlGetVersion) GetProcAddress(
```

Comment:
> I think that if checking for a feature is available we should do that.

But like, you don't have it. You're using undocumented methods and regkeys to do this that have absolutely zero guarantee that they will be there even after a GDR update, let alone a major OS release.